packages/components/src/components/data-list/data-list-cell.tsx

```diff
@@ -1,44 +1,53 @@
 import React from 'react';
 import classNames from 'classnames';
 import { isVanillaContract } from '@deriv/shared';
-import { TPassThrough } from './data-list';
+import { TPassThrough, TRow } from './data-list';
```

Comment:
Maybe it'll make sense if you transfer some frequently used types (e.g. `TPassThrough`) to `types/common.types.ts`? In case of `TPassThrough` it's imported in `data-list-cell.tsx` and `data-list-row.tsx`

Reply:
moved

Reply:
nope. only used in reports